content/docs/tasks/telemetry/kiali/index.md

```diff
@@ -0,0 +1,363 @@
+---
+title: Visualizing Your Mesh
+description: This task shows you how to visualize your services within an Istio mesh.
+weight: 49
+keywords: [telemetry,visualization]
+---
+
+This task shows you how to visualize different aspects of your Istio mesh.
+
+As part of this task, you will install the [Kiali](https://www.kiali.io) add-on
+and use the web-based graphical user interface for viewing service graphs of
+the mesh as well as your Istio configuration objects. You will also use the
+Kiali Public API to generate graph data in the form of consumable JSON.
+
+The [Bookinfo](/docs/examples/bookinfo/) sample application is used as the
+example application throughout this task.
+
+## Before you begin
+
+You first need to create a secret in your Istio namespace with credentials that
+users will use to authenticate to Kiali. See the
+[Helm README](https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/README.md#installing-the-chart)
+for details, but here is a command you can run to create a secret:
+
+```bash
+USERNAME=$(echo -n 'admin' | base64)
+PASSPHRASE=$(echo -n 'mysecret' | base64)
+NAMESPACE=istio-system
+kubectl create namespace $NAMESPACE
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: kiali
+  namespace: $NAMESPACE
+  labels:
+    app: kiali
+type: Opaque
+data:
+  username: $USERNAME
+  passphrase: $PASSPHRASE
+EOF
+```
+
+Once a Kiali secret is created, you can now install Kiali via Helm by following
+[these instructions](/docs/setup/kubernetes/helm-install/). Note that you need
+to use the `--set kiali.enabled=true` option when running the `helm` command.
+For example:
+
+{{< text bash >}}
+$ helm template --set kiali.enabled=true install/kubernetes/helm/istio --name istio --namespace istio-system > $HOME/istio.yaml
+$ kubectl apply -f $HOME/istio.yaml
+{{</ text >}}
+
+Once Istio and Kiali are installed, you should then deploy an application such
+as the [Bookinfo](/docs/examples/bookinfo/) sample application.
```

Comment:
Could we add a link to instructions for installing bookinfo?

Reply:
That /docs/examples/bookinfo does result in a link to the install instructions - it ends up giving the reader a link to this: https://istio.io/docs/examples/bookinfo/